packages/react-window-provider/README.md

```diff
@@ -0,0 +1,11 @@
+# @fluentui/react-window-provider
+
+**React Window Provider components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+
+These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+
+To import React Window Provider components:
+
+```js
+import { ComponentName } from '@fluentui/react-window-provider';
```

Comment:
needs some updates here :)

Reply:
whoops, my changes were not committed! Fixed.